### DISCUSSION TITLE
HOOK for pre kernel module loading?

### DISCUSSION THREAD

**Discussion by hongkongkiwi** [finit-project/finit#324] [Q&A] [answered]
Which hook is best to use for running a script before a kernel module loads?

I want to populate some files for a kernel module to use.

**Comment by troglobit** [comment, marked as answer]
Both the modules-load and modprobe plugins run at `HOOK_BASEFS_UP`, so I'd suggest the script hook right before that, `HOOK_MOUNT_POST`. I.e., `hook/mount/post`.
